packages/stark-testing/karma.conf.js

```diff
@@ -81,16 +54,50 @@ const rawKarmaConfig = {
 
 	// start these browsers
 	// available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-	browsers: ["Chrome"],
+	browsers: !!ciDetect() ? ["ChromeHeadlessNoSandbox"] : ["Chrome"],
 
 	// Continuous Integration mode
 	// if true, Karma captures browsers, runs the tests and exits
-	singleRun: false,
+	singleRun: !!ciDetect(),
+
+	// If true, tests restart automatically if a file is changed
+	restartOnFileChange: !ciDetect(),
 
 	// Timeout settings
 	browserNoActivityTimeout: 30000,
 	browserDisconnectTolerance: 1,
-	browserDisconnectTimeout: 30000
+	browserDisconnectTimeout: 30000,
+
+	// Configuration for coverage-istanbul reporter
+	coverageIstanbulReporter: {
```

Comment:
Mmm are you sure that running code coverage also in non-ci environment is fast enough? As far as I remember, this was a bit slower than normal testing without coverage, that's why we did it only for CI...

Wait... I see that the script passes `ng test --code-coverage` only for CI... so that is basically enabling code coverage only in CI right?

Reply:
Exactly! 😊 

The configuration for coverageIstanbulReporter is there but to enable this reporter you need the `--code-coverage` flag. It has no effect on `ng test` without the flag.